### PR TITLE
refactor: 💡 added my nfts to plug dropdown

### PR DESCRIPTION
**Pull request by OyinloluB**
## Why?

- [Description](https://www.notion.so/Post-Release-Feedback-4da676e97710421c8a19781d09f6dc93#5b5b6a65e0ab401f8ce021ed5623c1a8)

## How?

- Moved `myNfts` to plug dropdown menu

## Tickets?

- [Notion](https://www.notion.so/Post-Release-Feedback-4da676e97710421c8a19781d09f6dc93#5b5b6a65e0ab401f8ce021ed5623c1a8)

## Contribution checklist?

- [x] The commit messages are detailed
- [x] It does not break existing features (unless required)
- [x] I have performed a self-review of my own code
- [ ] Documentation has been updated to reflect the changes
- [ ] Tests have been added or updated to reflect the changes
- [x] All code formatting pass
- [x] All lints pass

## Demo?

https://user-images.githubusercontent.com/51888121/176232315-ea8089d3-5e47-4dd5-86bb-2e02bf60e3ed.mov

### CODE REVIEW

File: src/components/plug/plug-button.tsx [hunk 1]
```diff
@@ -80,6 +86,27 @@ export const PlugButton = ({
...
+    dispatch(filterActions.setMyNfts(!isMyNfts));
+    !filterExists(t('translation:buttons.action.myNfts'))
+      ? dispatch(
+          filterActions.applyFilter({
+            filterName: `${t('translation:buttons.action.myNfts')}`,
+            filterCategory: 'Display',
+          }),
+        )
+      : dispatch(
+          filterActions.removeFilter(
```
Comment (prasanthLalapeta) [inline src/components/plug/plug-button.tsx:104]: Let's not remove My NFT's filter when user clicks on My NFT's from plug dropdown

File: src/components/plug/plug-button.tsx [hunk 2]
```diff
@@ -120,6 +147,10 @@ export const PlugButton = ({
               <Icon icon="activity" paddingRight />
               <p>{t('translation:buttons.action.myActivity')}</p>
             </ListItem>
+            <ListItem onClick={setMyNfts}>
+              <Icon icon="myNfts" paddingRight />
```
Comment (prasanthLalapeta) [inline src/components/plug/plug-button.tsx:151]: I hope we verified MY NFT's logo with @rollsmorr1

File: src/components/plug/plug-button.tsx (hunk 1 above)
Comment (prasanthLalapeta) [reply src/components/plug/plug-button.tsx:104]: Irrespective of whether user selected MY NFT's filter or not from filters section, Let’s apply MY NFT's filter as soon as user clicks MY NFT's from plug dropdown

Comment (OyinloluB) [reply src/components/plug/plug-button.tsx]: https://user-images.githubusercontent.com/51888121/176394339-5904e51b-be71-4c1d-a354-1cebadfcfaa3.mov

**Review by prasanthLalapeta — APPROVED** [review]

**Pull request merged**